typescript-api/src/dancebox/interfaces/augment-api-consts.ts

```diff
@@ -24,324 +24,443 @@ export type __AugmentedConst<ApiType extends ApiTypes> = AugmentedConst<ApiType>
 declare module "@polkadot/api-base/types/consts" {
     interface AugmentedConsts<ApiType extends ApiTypes> {
         asyncBacking: {
-            /** Purely informative, but used by mocking tools like chospticks to allow knowing how to mock blocks */
+            /**
+             * Purely informative, but used by mocking tools like chospticks to allow knowing how to mock
+             * blocks
+             **/
             expectedBlockTime: u64 & AugmentedConst<ApiType>;
-            /** Generic const */
+            /**
+             * Generic const
+             **/
             [key: string]: Codec;
         };
         authorityMapping: {
             sessionRemovalBoundary: u32 & AugmentedConst<ApiType>;
-            /** Generic const */
+            /**
+             * Generic const
+             **/
             [key: string]: Codec;
         };
         balances: {
             /**
              * The minimum amount required to keep an account open. MUST BE GREATER THAN ZERO!
              *
-             * If you _really_ need it to be zero, you can enable the feature `insecure_zero_ed` for this pallet. However, you
-             * do so at your own risk: this will open up a major DoS vector. In case you have multiple sources of provider
-             * references, you may also get unexpected behaviour if you set this to zero.
+             * If you *really* need it to be zero, you can enable the feature `insecure_zero_ed` for
```

Comment:
Why did this change? Because of a polkadot js update? If so, then a separate PR with only the upgrade will make this one smaller in size eventually

Reply:
This was caused by biome formatting our comments.